de/news/_posts/2017-04-14-deutsche-ruby-mailingliste.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: news_post
+title: "Deutschsprachiges Ruby-Forum schließt und wird durch Mailingliste ersetzt"
```

Comment:
Maybe something more "positive" and from the point of view of ruby-lang.org (since the Ruby-Forum is not directly related to "us"), like simply _Neue deutschspr. Maliingliste ruby-de [eingerichtet]_ or similar?

Just a thought.